spring-cloud-task-integration-tests/src/test/java/org/springframework/cloud/task/executionid/TaskStartTests.java

```diff
@@ -207,6 +247,16 @@ private boolean waitForDBToBePopulated() throws Exception {
 		return isDbPopulated;
 	}
 
+	private void enableFooLock() {
+		SimpleJdbcInsert taskLockInsert = new SimpleJdbcInsert(this.dataSource).withTableName("TASK_LOCK");
+		Map<String, Object> taskLockParams = new HashMap<>();
+		taskLockParams.put("LOCK_KEY", "acbd18db-4cc2-385c-adef-654fccc4a4d8");
+		taskLockParams.put("REGION", "DEFAULT");
+		taskLockParams.put("CLIENT_ID", "ac39193d-4a24-4112-9d1b-5d79d6286921");
```

Comment:
Is there a reason these UUIDs are hardcoded?

Reply:
The lock key is generated on the task name `foo`.  And that is how it keyed.  as far as the client id, it was just an id that was used.

Reply:
I’d just expect to use `UUID.randomUUID()` to prevent the potential for duplicates across test runs.

Reply:
The lock key is created using SI's UUIDConverter, which will generate a UUID from a key passed in.  I will update the test so that it will accept a key and create its UUID from it.